packages/pigeon/tool/run_tests.dart

```diff
@@ -138,6 +136,17 @@ Future<void> _validateGeneratedExampleFiles() async {
   exit(1);
 }
 
+Set<String> _extensionsForLanguage(GeneratorLanguage language) {
+  return switch (language) {
+    GeneratorLanguage.cpp => <String>{'cc', 'cpp', 'h'},
+    GeneratorLanguage.dart => <String>{'dart'},
+    GeneratorLanguage.java => <String>{'java'},
+    GeneratorLanguage.kotlin => <String>{'kt'},
+    GeneratorLanguage.swift => <String>{'swift'},
+    GeneratorLanguage.objc => <String>{'m', 'mm'},
```

Comment:
how are we handling .h files for objc?

Reply:
Oops! They should be listed here; having duplicates will be fine since.